reference/android-features.md

```diff
@@ -16,7 +16,7 @@ Anbox Cloud implements support for various Android features. The following table
 | WiFi               |  ✓  | Only simulated WiFi is provided to the Android instance. |
 | [Multi-Display](https://source.android.com/devices/tech/display/multi_display) | | |
 | [Data use](https://source.android.com/devices/tech/datausage)| | |
-| Telephony / mobile connectivity | | |
+| Telephony / mobile connectivity | ✓ | |
```

Comment:
I mean... This is still only partial support, right? We don't support all features, we don't support calls, etc.